Application/Api/Education/Certificate/Generator/Repository/BfsHj.php

```diff
@@ -0,0 +1,930 @@
+            )
+            ->addElementColumn((new Element())
+                ->setContent('&nbsp')
+                , '40%'
+            )
+            ->addElementColumn((new Element())
+                ->setContent('Schuljahr {{ Content.P' . $personId . '.Division.Data.Year }}')
+                ->styleAlignCenter()
+                ->styleBorderBottom('0.5px', '#000', 'dotted')
+                , '30%'
+            )
+        );
+
+        $Slice->addElement((new Element())
+            ->setContent('
+            {% if(Content.P'.$personId.'.Person.Data.Name.Salutation is not empty) %}
+                {{ Content.P'.$personId.'.Person.Data.Name.Salutation }}
+            {% else %}
+                Frau/Herr
+            {% endif %}
+            {{ Content.P' . $personId . '.Person.Data.Name.First }}
+            {{ Content.P' . $personId . '.Person.Data.Name.Last }}')
+//            ->setContent('
+//            {% if Content.P' . $personId . '.Person.Common.BirthDates.Gender == 2 %}
+//                Frau
+//            {% else %}
+//                {% if Content.P' . $personId . '.Person.Common.BirthDates.Gender == 1 %}
+//                    Herr
+//                {% else %}
+//                    Frau/Herr
+//                {% endif %}
+//            {% endif %}
+//            {{ Content.P' . $personId . '.Person.Data.Name.First }}
+//            {{ Content.P' . $personId . '.Person.Data.Name.Last }}')
+            ->styleBorderBottom('0.5px', '#000', 'dotted')
+            ->styleAlignCenter()
+            ->styleTextSize('26px')
+            ->stylePaddingTop('20px')
+            ->styleMarginBottom('20px')
+        );
+
+        $Slice->addSection((new Section())
+            ->addElementColumn((new Element())
+                ->setContent('geboren am  {{ Content.P' . $personId . '.Person.Common.BirthDates.Birthday }}')
+                ->styleAlignCenter()
+                ->styleBorderBottom('0.5px', '#BBB')
+                , '30%'
+            )
+            ->addElementColumn((new Element())
+                ->setContent('&nbsp')
+                , '40%'
+            )
+            ->addElementColumn((new Element())
+                ->setContent('in {{ Content.P' . $personId . '.Person.Common.BirthDates.Birthplace }}')
+                ->styleAlignCenter()
+                ->styleBorderBottom('0.5px', '#BBB')
+                , '30%'
+            )
+        );
+
+        $Slice->addElement((new Element())
+            ->setContent('hat im zurückliegenden ' . $period . ' folgende Leistungen erreicht:')
+            ->styleAlignCenter()
+            ->styleTextSize('16px')
+            ->stylePaddingTop('20px')
+            ->styleBorderBottom('0.5px', '#BBB')
+        );
+
+        return $Slice;
+    }
+
+    /**
+     * @param $personId
+     * @param TblCertificate $tblCertificate
+     *
+     * @return Slice
+     */
+    public function getSubjectLineAcross($personId, TblCertificate $tblCertificate)
```

Comment:
Codacy found an issue: [The method getSubjectLineAcross() has an NPath complexity of 505. The configured NPath complexity threshold is 200.](https://app.codacy.com/manual/gerdchristian-kunze/SPHERE-Framework/pullRequest?prid=5757982)